src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+import core.basesyntax.figures.Figure;
+import core.basesyntax.figures.FigureSupplier;
+import core.basesyntax.figures.Rectangle;
+
+public class Main {
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+
+        Figure[] figures = new Figure[]{
+                figureSupplier.getRandomFigure(),
+                figureSupplier.getRandomFigure(),
+                figureSupplier.getRandomFigure(),
+                figureSupplier.getDefaultFigure(),
+                new Rectangle(10, 20, "BLUE")
+                };
```

Comment:
you need to create 3 random and 3 default figures. Can you use a loop for this? Why did you add ```new Rectangle(10, 20, "BLUE")```?